src/dashboard/src/media/js/ingest.js

```diff
@@ -294,22 +294,28 @@ $(function()
 
           var chainId = $select.find('option:selected').val();
           var unitId = this.model.sip.get('uuid');
```

Comment:
Thanks @sevein. In response to your other message about how you'd handle a potential exception here, I don't think we need to do that at this point. You're right that it seems highly unlikely.